### PR TITLE
3.5.4 changelog

### PR DESCRIPTION
**Pull request by jeanbisutti**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # CHANGELOG
 
+Application Insights Java 3.5.4 GA (07/29/2024)
+
+### Bug fixes
+* Fix possible OOM with Wildfly
```
Comment (trask) [inline CHANGELOG.md:6]:
```suggestion
* Fix possible OOM with Wildfly when agent debug logging is enabled
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+([#3796](https://github.com/microsoft/ApplicationInsights-Java/pull/3796))
+
+### Enhancements:
+* Fail to start-up if proxy host contains http or https
```
Comment (trask) [inline CHANGELOG.md:10]:
```suggestion
* Validate proxy host configuration on startup
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
+([#3776](https://github.com/microsoft/ApplicationInsights-Java/pull/3676))
+* Help the user to configure the connection string
+([#3788](https://github.com/microsoft/ApplicationInsights-Java/pull/3788))
+
+### CVE
+* Fix CVE-2024-35255
```
Comment (trask) [inline CHANGELOG.md:16]:
```suggestion
* Update azure-identity (CVE-2024-35255)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,21 @@
...
+Application Insights Java 3.5.4 GA (07/29/2024)
+
+### Bug fixes
+* Fix possible OOM with Wildfly
+([#3796](https://github.com/microsoft/ApplicationInsights-Java/pull/3796))
+
+### Enhancements:
+* Fail to start-up if proxy host contains http or https
+([#3776](https://github.com/microsoft/ApplicationInsights-Java/pull/3676))
+* Help the user to configure the connection string
```
Comment (trask) [inline CHANGELOG.md:12]:
```suggestion
* Better error message when connection string is not configured
```

**Review by trask — APPROVED** [review]

**Comment by heyams** [comment]
can you mention https://github.com/microsoft/ApplicationInsights-Java/pull/3765?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,21 @@
...
+([#3788](https://github.com/microsoft/ApplicationInsights-Java/pull/3788))
```
Comment (heyams) [inline CHANGELOG.md:13]:
```suggestion
([#3788](https://github.com/microsoft/ApplicationInsights-Java/pull/3788))
* Update browser snippet version
([#3765](https://github.com/microsoft/ApplicationInsights-Java/pull/3765))
```

**Review by heyams — APPROVED** [review]

**Pull request merged**